src/tiler/package.json

```diff
@@ -6,8 +6,12 @@
   "dependencies": {
     "windshaft": "0.36.0"
   },
+  "devDependencies": {
+    "supervisor": "^0.6.0"
```

Comment:
A thing I've wanted to evaluate to try and solve this problem in a more general way, but haven't had time yet is [Watchman](https://facebook.github.io/watchman/).